src/core/public/chrome/ui/header/home_icon.test.tsx

```diff
@@ -4,251 +4,109 @@
  */
 
 import React from 'react';
-import { mountWithIntl } from 'test_utils/enzyme_helpers';
-import { DEFAULT_MARK, DEFAULT_DARK_MARK, HomeIcon } from './home_icon';
-
-// TODO: many of these tests cover the conditional logic of which mark to load depending on raw branding configurations
-// Dark mode and custom/default fallbacks should instead be consolidated and centralized elsewhere:
-// https://github.com/opensearch-project/OpenSearch-Dashboards/issues/895#issuecomment-1164995007
-describe('Home button icon ', () => {
-  describe('in condensed light mode ', () => {
-    it('uses opensearch mark if no mark provided', () => {
-      const branding = {
-        darkMode: false,
-        logo: {},
-        mark: {},
-        applicationTitle: 'custom title',
-        assetFolderUrl: 'base/ui/default_branding',
-        useExpandedHeader: false,
-      };
-      const component = mountWithIntl(<HomeIcon {...branding} />);
-      const icon = component.find('EuiIcon');
-      expect(icon.prop('type')).toEqual(`${branding.assetFolderUrl}/${DEFAULT_MARK}`);
-      expect(icon.prop('title')).toEqual(`${branding.applicationTitle} home`);
-      expect(component).toMatchSnapshot();
-    });
+import { shallow } from 'enzyme';
+import { HomeIcon } from './home_icon';
+import { getLogosMock } from '../../../../common/mocks';
 
-    it('uses opensearch mark if custom logo provided without mark', () => {
-      const branding = {
-        darkMode: false,
-        logo: { defaultUrl: '/defaultModeLogo' },
-        mark: {},
-        applicationTitle: 'custom title',
-        assetFolderUrl: 'base/ui/default_branding',
-        useExpandedHeader: false,
-      };
-      const component = mountWithIntl(<HomeIcon {...branding} />);
-      const icon = component.find('EuiIcon');
-      expect(icon.prop('type')).toEqual(`${branding.assetFolderUrl}/${DEFAULT_MARK}`);
-      expect(icon.prop('title')).toEqual(`${branding.applicationTitle} home`);
-      expect(component).toMatchSnapshot();
-    });
+const mockTitle = 'Page Title';
 
-    it('uses custom mark default mode URL', () => {
-      const branding = {
-        darkMode: false,
-        logo: {},
-        mark: { defaultUrl: '/defaultModeMark' },
-        applicationTitle: 'custom title',
-        assetFolderUrl: 'base/ui/default_branding',
-        useExpandedHeader: false,
-      };
-      const component = mountWithIntl(<HomeIcon {...branding} />);
-      const icon = component.find('EuiIcon');
-      expect(icon.prop('type')).toEqual(branding.mark.defaultUrl);
-      expect(icon.prop('title')).toEqual(`${branding.applicationTitle} home`);
-      expect(component).toMatchSnapshot();
+describe('Home icon', () => {
+  describe('Unbranded', () => {
```

Comment:
super minor nit - lowercasing these second level `describe` strings would makes the snapshots more readable